[Excel] (Custom Functions) Addressing feedback from Customer Issue (#746)
OfficeDev/office-js-docs-pr: docs/tutorials/excel-tutorial-create-custom-functions.md

This command starts the web server, and sideloads your custom function add-in into Excel for Windows.

> [!NOTE]

> > If your add-in does not load, check that you have completed step 3 properly. You can also enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues, as well.

Reviewer: ```suggestion
> If your add-in does not load, check that you have completed step 3 properly. You can also enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as any installation or runtime problems. Runtime logging writes `console.log` statements to a log file to help you find and fix issues.
```

Reviewer: Some suggested rewrites. Hope this works.

Author: cool, I will commit those. @davidchesnut  - can you also make the rewrites to the main trouble shooting guide, I had copied the blurb from there.
